src/README.md

```diff
@@ -0,0 +1,30 @@
+# Command line apps in Rust
+
+Rust is a statically compiled, fast language with great tooling and a rapidly growing ecosystem.
```

Comment:
I have never seen a language not call itself "fast", even if this was not the case :P 

But besides that, I would turn the sentence around:

"Rust is a fast, statically compiled language, with great tooling..."

Feels more natural IMO

Reply:
Good point. Another idea: How about using some of that new marketing lingo?

> Rust is a statically compiled language that values performance, reliability, and developer productivity.

Reply:
I didn't know that was the new marketing lingo, but I like it :ok_hand:

Reply:
I think the best way is to prove that it is fast rather than calling itself fast, it would be nice to show how zero-cost abstractions apply here, I seen that appeared everywhere but not sure how does it affect rust cli apps, and a simple benchmark can be a plus.